### PR TITLE
New version: Rime.Weasel version 0.15.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136425)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-25T21:16:41.1427396Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[AppsAndFeaturesEntries][0] Description: Value type not permitted by 'type' constraint. Error context: <root>[AppsAndFeaturesEntries] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'AppsAndFeaturesEntries'. ] File: Rime.Weasel.installer.yaml Manifest Error: Encountered unexpected root node.

(Automated response - build 660.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-25T21:16:41.1427396Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[AppsAndFeaturesEntries][0] Description: Value type not permitted by 'type' constraint. Error context: <root>[AppsAndFeaturesEntries] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'AppsAndFeaturesEntries'. ] File: Rime.Weasel.installer.yaml Manifest Error: Encountered unexpected root node.

(Automated response - build 660.)

File: manifests/r/Rime/Weasel/0.15.0/Rime.Weasel.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+PackageIdentifier: Rime.Weasel
+PackageVersion: 0.15.0
+InstallerType: nullsoft
+ProductCode: Weasel
+ReleaseDate: 2023-06-06
+InstallerSwitches:
+ Silent: /S
+ SilentWithProgress: /S
+AppsAndFeaturesEntries:
+- {}
```
Comment (stephengillie) [inline manifests/r/Rime/Weasel/0.15.0/Rime.Weasel.installer.yaml:13]:
```suggestion
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-25T21:16:41.1427396Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[AppsAndFeaturesEntries][0] Description: Value type not permitted by 'type' constraint. Error context: <root>[AppsAndFeaturesEntries] Description: Failed to validate item #0 in array. Error context: <root> Description: Failed to validate against schema associated with property name 'AppsAndFeaturesEntries'. ] File: Rime.Weasel.installer.yaml Manifest Error: Encountered unexpected root node.

(Automated response - build 660.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 661.)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

The installation isn't unattended. Is there an installer switch to have the package install silently?

(Automated response - build 661.)

**Pull request closed without merging**